src/AzureAuth/Commands/CommandAzureAuth.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth.Commands
+{
+    using McMaster.Extensions.CommandLineUtils;
+
+    /// <summary>
+    /// The command main class parses commands and dispatches to the corresponding methods.
+    /// </summary>
+    [Command(Name = "azureauth", Description = "A CLI interface to MSAL (Microsoft Authentication Library)")]
+    [Subcommand(typeof(CommandAad), typeof(CommandAdo))]

```

Comment:
Just curious here. Why use
```csharp
[Subcommand(typeof(Ado.CommandPat))]
[Subcommand(typeof(Ado.CommandToken))]
```
in `CommandAdo`, but use
```csharp
[Subcommand(typeof(CommandAad), typeof(CommandAdo))]
```
here? Is one preferable over the other? Would it help us to be consistent?

Reply:
Ah good catch, I will update the other to use individual rows. I prefer individual attributes, because they can be indiependetly added, removed, or re-ordered line by line, instead of having to edit the inline order, and manage commas.